ci/gpuci/run.sh

```diff
@@ -21,9 +21,23 @@ echo $DH_TOKEN | docker login --username $DH_USER --password-stdin &> /dev/null
 BUILD_TAG="${RAPIDS_VER}-cuda${CUDA_VER}-devel-${LINUX_VER}-py${PYTHON_VER}"
 
 # Setup BUILD_ARGS
+case $RAPIDS_VER in
```

Comment:
When we need to update, is this the only section which requires modification ?

Reply:
This and the `RAPIDS_VER` block in the axis file:

https://github.com/rapidsai/dask-build-environment/blob/79e1b8bd794d37f24aede7ed1aa28b32ec3d4a47/ci/axis/dask.yaml#L20-L22